docs/graph/PowerBIEmbedded.md

```diff
@@ -0,0 +1,82 @@
+---
+title: PowerBIEmbedded Control
+author: OGcanviz
+description: The PowerBIEmbedded control is a simple wrapper to an IFRAME for a PowerBI embed.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, PowerBIEmbedded Control
+---
+
+# PowerBIEmbedded Control
+
+The [PowerBIEmbedded Control](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.graph.powerbiembedded) is a simple wrapper to an IFRAME for a PowerBI embed.
+
+## Syntax
+
+```xaml
+<Page ...
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.Graph"/>
+
+<controls:PowerBIEmbedded x:Name="PowerBIEmbedded1"
+    ClientId="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+    GroupId="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" />
+```
+
+## Example Image
+
+![PowerBIEmbedded animation](../resources/images/Graph/PowerBIEmbedded.png)
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| ClientId | String | Gets or sets the client id of Azure AD app registration (v1) |
+| GroupId | String | Gets or sets the identifier of a Group. Optional when EmbedUrl is specified, if this is used, a dropdown will appear that lets users select amongst different reports in a Group |
+| EmbedUrl | String | Gets or sets the Url of the embed, optional when GroupId is specified |
+
+## Integration
```

Comment:
This is showing weird in the sample app:

![image](https://user-images.githubusercontent.com/24302614/42787654-5682acce-8910-11e8-985b-e2be84dce357.png)

Reply:
duplicate